web/docs/understand/formats/zeek.md

```diff
@@ -62,8 +62,15 @@ You can import this log as follows:
 vast import zeek < conn.log
 ```
 
+:::info type mapping
+The `import zeek` command implicitly
+[aliases](/docs/understand/data-model/type-system.md) the types `count`, `real`,
+and `addr` from the Zeek TSV `#types` header line to VAST's basic types
+`uint64`, `double`, and `ip`.
+:::
+
 When Zeek [rotates logs][zeek-log-rotation], it produces compressed batches of
-`*.tar.gz` regularly. If log freshness is not a priority, you could trigger an
+`*.log.gz` regularly. If log freshness is not a priority, you could trigger an
```

Comment:
```suggestion
`*.tar.gz` regularly. If log freshness is not a priority, you could trigger an
```

I believe this was already correct.

Reply:
Zeek creates `gz` not `tar.gz`.

Reply:
Nevermind then, I must've misremembered.